scanners/ios/podfile.go

```diff
@@ -148,18 +150,18 @@ begin
 	# For example: require_relative '../node_modules/react-native/scripts/react_native_pods'
 	Dir.chdir(File.dirname(podfile_path))
 	podfile = Pod::Podfile.from_file(podfile_path)
-	pth = podfile.workspace_path
-	puts "#{{ :data => pth }.to_json}"
+	if !podfile.workspace_path
+		puts "#{{ :data => "" }.to_json}"
```

Comment:
why not use the same as in the case of `getTargetDefinitionProjectMap`:

```
	if !targets
		puts "#{{}.to_json}"
		exit
	end
```

Reply:
updated